src/store/actions/search.js

```diff
@@ -75,3 +75,19 @@ export const searchPaginationChange = (userIp, userAgent, limit, sortBy, current
     dispatch(searchPaginationChangeDone(start, currentPage));
   }
 };
+
+export const searchSortByChangeDone = (sortBy) => {
+  return {
+    type: actionTypes.SEARCH_SORT_BY_CHANGE_DONE,
+    sortBy: sortBy
+  }
+};
+
+export const searchSortByChange = (userIp, userAgent, limit, sortBy, start, query, location, country, radius, jobType, age) => {
```

Comment:
For large lists of parameters like this, I recommend object parameters. it could either be a big one to catch all criteria, such as:

```
export const searchSortByChange = ({userIp, userAgent, limit, sortBy, start, query, location, country, radius, jobType, age})
```

or you could group them, for example:

```
export const searchSortByChange = ({userIp, userAgent}, {limit, sortBy, start, query}, {location, country, radius}, {jobType, age})
```

this also means, here you don't have to actually destructure, as `searchGo` is doing the heavy lifting:

```
export const searchSortByChange = (criteria) => (dispatch) => {
  const { sortBy } = criteria;
  dispatch(searchGo(criteria));
  dispatch(searchSortByChangeDone(sortBy));
}
```

In general, the fewer number of arguments to a function, the better.